README.md

```diff
@@ -48,9 +48,10 @@ pushed to your docker repository.
 Use the appropriate make targets or scripts in `hack`:
 
 - `make dev`: Deploys the serverless-operator without deploying Knative Serving, Eventing and Kafka components.
-- `make install`: Scales the cluster appropriately, deploys serverless-operator, Knative Serving and Eventing.
+- `make install`: Scales the cluster appropriately, deploys serverless-operator, Knative Serving, Eventing and Kafka components.
 - `make install-serving`: Scales the cluster appropriately, deploys serverless-operator and Knative Serving.
 - `make install-eventing`: Scales the cluster appropriately, deploys serverless-operator and Knative Eventing.
+- `make install-kafka`: Scales the cluster appropriately, deploys serverless-operator and Knative Kafka components.
```

Comment:
and strimzi?

Reply:
I don't see this makefile target anymore in master -- I'll drop from my PR as well